pysyncthru/tests/test_structure/syncthru_mock_server.py

```diff
@@ -17,21 +13,21 @@ class SyncThruServer(HTTPServer):
 
     blocked = False
 
-    def set_blocked(self):
+    def set_blocked(self) -> None:
         self.blocked = True
 
-    def unset_blocked(self):
+    def unset_blocked(self) -> None:
         self.blocked = False
 
 
 class SyncThruRequestHandler(SimpleHTTPRequestHandler):
-    def do_GET(self):
-        if self.server.blocked:
+    def do_GET(self) -> None:
+        if cast(SyncThruServer, self.server).blocked:
```

Comment:
Rather than simply casting to `SyncThruServer` we might overwrite the `__init__` method for type hinting or even throw an error if any server is passed that is not a `SyncThruServer`

Reply:
`__init__` added.